crates/sui-sdk/examples/coin_read_api.rs

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+mod utils;
+
+use futures::{future, stream::StreamExt};
+use sui_sdk::SuiClientBuilder;
+use utils::sui_address_for_examples;
+
+#[tokio::main]
+async fn main() -> Result<(), anyhow::Error> {
+    let sui = SuiClientBuilder::default().build_testnet().await?; // testnet Sui network
+    println!("Sui testnet version{:?}", sui.api_version());
+    // create a random Sui address for examples.
+    // Check utils module if you want to use a local wallet, or use SuiAddress::from_str("sui_address") for a specific address
+    let active_address = sui_address_for_examples().await?;
+
+    // ************ COIN READ API ************ //
+
+    // Get coins
+    let coins = sui
+        .coin_read_api()
+        .get_coins(active_address, None, None, Some(5)) // get the first five coins. Note that this can be filtered by coin type: coin_type: Some("0x2::sui::SUI".to_string())
```

Comment:
nit: I'm not sure whether `cargo fmt` will help with this, but can we limit the line length to either 80 or 100 characters? (Whatever is the limit we use throughout the rest of the codebase -- I think it's 80).